scaleway/resource_container.go

```diff
@@ -313,7 +313,7 @@ func resourceScalewayContainerUpdate(ctx context.Context, d *schema.ResourceData
 	}
 
 	if d.HasChanges("description") {
-		req.Description = expandStringPtr(d.Get("description"))
+		req.Description = expandUpdatedStringPtr(d.Get("description"))
```

Comment:
should we add an `hasChange = true` here?

Reply:
Only one update request can be made in this function, if update was triggered, it is because at least one field has changed